scripts/metavar.py

```diff
@@ -1823,6 +1865,32 @@ def __eq__(self, other):
         list equality"""
         return self is other
 
+    @classmethod
+    def unit_cnv_match(cls, hunits, var):
+        """Return VarUnitConv if <hunits> and <var> require unit
+        conversion, otherwise, return None"""
+        ucmatch = {"from": '' ,"to": ''}
+        function_name_from = '{0}__to__{1}'.format(hunits, var.get_prop_value('units'))
+        function_name_to   = '{0}__to__{1}'.format(var.get_prop_value('units'), hunits)
+
+        try:
+            function_from   = getattr(unit_conversion, function_name_from)
+            ucmatch["from"] = VarUnitConv(function_from(),key="from",
+                                          local_name=var.get_prop_value('local_name'),
+                                          kind=var.get_prop_value('kind'))
+        except:
+            ucmatch["from"] = None
+        try:
+            if (var.get_prop_value('intent') != 'in'):
+                function_to   = getattr(unit_conversion, function_name_to)
+                ucmatch["to"] = VarUnitConv(function_to(),key="to",
+                                            local_name=var.get_prop_value('local_name'),
+                                            kind=var.get_prop_value('kind'))
+        except:
+            ucmatch["to"]   = None
```

Comment:
Ditto: I think we want to generate an error and abort if a requested unit conversion is not implemented.